transforms/ember-object/__testfixtures__/decorators.output.js

```diff
@@ -28,6 +29,9 @@ class Foo extends EmberObject {
     return "abc";
   }
 
+  @action
+  someActionUtil;
```

Comment:
This syntax won't work, it'll just declare an empty field. You'd need to do

```js
@action
someActionUtil = someActionUtil;
```

but that also won't work because of the way `@action` works. How common is this pattern?

I think this may be the best solution:

```js
@action
someActionUtil() {
  return someActionUtil.call(this, ...arguments);
}
```

Reply:
Oh ok! Wouldn't it be considered as `shorthand` or classes do not support that?

Anyway, we have quite few occurrences of this pattern from what I have seen and I think it should be fixed. I'll update the PR.

Reply:
It would normally be restructuring, but that only works with objects. Classes definitions do not support it, unfortunately 😕